README.md

```diff
@@ -1064,545 +983,270 @@ helpLinks: {
 }
 ```
 
-- **helpLinks.help** - Custom link href for the "Help" link
+##### helpLinks.help
 
-- **helpLinks.forgotPassword** - Custom link href for the "Forgot Password" link
+Custom link href for the "Help" link
 
-- **helpLinks.unlock** - Custom link href for the "Unlock Account" link. For this link to display, `features.selfServiceUnlock` must be set to `true`, and the self service unlock feature must be enabled in your admin settings.
+##### helpLinks.forgotPassword
 
-- **helpLinks.factorPage** - Custom link object `{text, href}` that will be added to all MFA pages.
+Custom link href for the "Forgot Password" link
 
-- **helpLinks.custom** - Array of custom link objects `{text, href, target}` that will be added to the *"Need help signing in?"* section. The `target` of the link is optional.
+##### helpLinks.unlock
 
-#### Sign Out Link
+Custom link href for the "Unlock Account" link. For this link to display, `features.selfServiceUnlock` must be set to `true`, and the self service unlock feature must be enabled in your admin settings.
 
-Set the following config option to override the sign out link URL. If not provided, the widget will navigate to Primary Auth.
-
-```javascript
-signOutLink: 'https://www.signmeout.com'
-```
+##### helpLinks.custom
 
-### Buttons
+Array of custom link objects `{text, href, target}` that will be added to the *"Need help signing in?"* section. The `target` of the link is optional.
 
-You can add buttons to the Primary Auth page by setting the following config options.
+- **features.autoFocus** - Automatically focuses the first input field of any form when displayed. Defaults to `true`.
 
-#### Custom Buttons
+### Hooks
 
-You can add custom buttons underneath the login form on the primary auth page by setting the following config options. If you'd like to change the divider text, use the `i18n` config option.
+Asynchronous callbacks can be invoked before or after a specific view is rendered. Hooks can be used to add custom logic such as tracking, logging, or additional user input. Normal execution is blocked while the hooks is executing and will resume after the Promise returned from the hook function resolves. Hooks can be added via config, as shown below, or at runtime using the [before](#before) or [after](#after) methods. The full list of views can be found in [RemediationConstants.js](https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/RemediationConstants.js#L19).
```

Comment:
should we use word `tracking` ?

Reply:
`the hooks are executing`

Reply:
maybe "analytics" or "instrumentation" would be a better euphemism here